<!DOCTYPE html>
<html style="font-size: 16px;" lang="en"><head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="keywords" content="Strand Bias, Results of analysis for different datasets, Datasets">
    <meta name="description" content="">
    <title>Home</title>
    <link rel="stylesheet" href="{{ url_for('static', filename='styles/nicepage.css')}}" media="screen">
    <link rel="stylesheet" href="{{ url_for('static', filename='styles/Home.css')}}" media="screen">

    <script class="u-script" type="text/javascript" src="{{ url_for('static',filename='styles/jquery.js') }}" defer=""></script>
    <script class="u-script" type="text/javascript" src="{{ url_for('static',filename='styles/nicepage.js') }}" defer=""></script>
    <meta name="generator" content="Nicepage 4.19.3, nicepage.com">
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,500,500i,600,600i,700,700i,800,800i">
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": ""
}</script>
    <meta name="theme-color" content="#478ac9">
    <meta property="og:title" content="Home">
    <meta property="og:type" content="website">
  </head>
  <body class="u-body u-xl-mode" data-lang="en"><header class="u-clearfix u-header u-palette-3-light-3 u-header" id="sec-0286"><div class="u-clearfix u-sheet u-valign-middle u-sheet-1">

        <nav class="u-menu u-menu-one-level u-offcanvas u-menu-1" data-responsive-from="MD">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px; font-weight: 700; text-transform: uppercase;">
            <a class="u-button-style u-custom-active-border-color u-custom-active-color u-custom-border u-custom-border-color u-custom-borders u-custom-hover-border-color u-custom-hover-color u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-active-color u-custom-text-color u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg class="u-svg-link" viewBox="0 0 24 24"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg class="u-svg-content" version="1.1" id="menu-hamburger" viewBox="0 0 16 16" x="0px" y="0px" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg"><g><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</g></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-spacing-30 u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('home_page') }}" style="padding: 10px 0px;">Home</a>
</li><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('aproject_page') }}" style="padding: 10px 0px;">About Project</a>
</li><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('datasets_page') }}" style="padding: 10px 0px;">Datasets</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-inner-container-layout u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('home_page') }}">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('project_page') }}">About Project</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('datasets_page') }}">Datasets</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('analysis_page') }}">Analysis</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header>
<!--    <section class="u-clearfix u-palette-3-light-2 u-section-1" id="sec-0fbe">-->
<!--      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">-->
<!--        <h3 class="u-text u-text-1">IT Solution &amp; IT Startup</h3>-->
<!--        <p class="u-text u-text-2">Strand bias is a situation when one DNA strand is favoured over the other strand.-->
<!--            Due to the varying amount of evidence favouring one gene variant compared to another, there is a possibility-->
<!--            of an incorrect evaluation of the result. Knowing the correct frequencies of gene variants is crucial for-->
<!--            population genotype analyses. It is also an important factor in medicine because frequencies of gene variants-->
<!--            identify numerous genetic disorders. In theory, the incorrect evaluation of results could lead to inaccurate-->
<!--            diagnosis and, therefore, wrong treatment.</p>-->
<!--<p class="u-text u-text-2">-->
<!-- In bioinformatics, k-mers refer to all possible substrings of length k contained in the nucleotide sequence.-->
<!--    Reverse complement of k-mer is created by constructing a complementary sequence to it and then reversing the-->
<!--    sequence so that the last nucleotide becomes the first. Here we recognise strand bias as varying frequencies of-->
<!--    the k-mers and their reverse complements. When both k-mer and its reverse complement have the same number of-->
<!--    occurrences, it means this k-mer has no bias. However, when their frequencies vary, it means k-mer has strand bias.-->
<!--    The higher the difference between k-mer and its reverse complement, the more concerning the strand bias is.-->
<!--</p>-->
<!--          <p class="u-text u-text-2">-->
<!--In the interactive plots, strand bias is represented by the ratio of less frequent (LF) of the k-mer and its complement-->
<!--              towards the more frequent (MF) subtracted from 1 and converted into a percentage. The subtraction from 1-->
<!--               is needed because a ratio represents how much the strings are ‘alike’, while the focus should be on how-->
<!--              much they differ. </p>-->
<!--          <p class="u-text u-text-2" align="center">-->
<!--      </div>-->
<!--    </section>-->
<!--    <section class="u-align-left u-clearfix u-palette-3-light-3 u-section-2" id="sec-64ec">-->
<!--      <div class="u-clearfix u-sheet u-sheet-1">-->
<!--        <div class="fr-view u-clearfix u-rich-text u-text u-text-1">-->
<!--          <h3 style="text-align: left;">Strand Bias</h3>-->
<!--                    <p style="text-align: left;">-->
<!--            <span style="line-height: 2.0;">-->
<!--                Strand bias is a situation when one DNA strand is favoured over the other strand.-->
<!--            Due to the varying amount of evidence favouring one gene variant compared to another, there is a possibility-->
<!--            of an incorrect evaluation of the result. Knowing the correct frequencies of gene variants is crucial for-->
<!--            population genotype analyses. It is also an important factor in medicine because frequencies of gene variants-->
<!--            identify numerous genetic disorders. In theory, the incorrect evaluation of results could lead to inaccurate-->
<!--                diagnosis and, therefore, wrong treatment.-->
<!--         &nbsp;</span>-->
<!--          </p>-->
<!--            <p style="text-align: left;">-->
<!--            <span style="line-height: 2.0;">-->
<!--                In bioinformatics, k-mers refer to all possible substrings of length k contained in the nucleotide sequence.-->
<!--    Reverse complement of k-mer is created by constructing a complementary sequence to it and then reversing the-->
<!--    sequence so that the last nucleotide becomes the first. Here we recognise strand bias as varying frequencies of-->
<!--    the k-mers and their reverse complements. When both k-mer and its reverse complement have the same number of-->
<!--    occurrences, it means this k-mer has no bias. However, when their frequencies vary, it means k-mer has strand bias.-->
<!--    The higher the difference between k-mer and its reverse complement, the more concerning the strand bias is.-->

<!--         &nbsp;</span>-->
<!--          </p>-->
<!--            <p style="text-align: left;">-->
<!--            <span style="line-height: 2.0;">-->
<!--In the interactive plots, strand bias is represented by the ratio of less frequent (LF) of the k-mer and its complement-->
<!--              towards the more frequent (MF) subtracted from 1 and converted into a percentage. The subtraction from 1-->
<!--               is needed because a ratio represents how much the strings are ‘alike’, while the focus should be on how-->
<!--              much they differ.-->
<!--         &nbsp;</span>-->
<!--          </p>-->
<!--            <p style="text-align: center;">-->
<!--            <span style="line-height: 2.0;">-->
<!--                SB = [(1 − LF/MF) ∗ 100]&nbsp;</span>-->
<!--          </p>-->
<!--          <h2 style="text-align: left;">Results of analysis for different datasets</h2>-->
<!--          <p style="text-align: left;">-->
<!--            <span style="line-height: 2.0;">Morbi enim nunc faucibus a pellentesque sit amet porttitor. Mattis pellentesque id nibh tortor id aliquet lectus proin nibh. Molestie nunc non blandit massa enim nec dui. Felis imperdiet proin fermentum leo vel orci porta. Natoque penatibus et magnis dis parturient montes nascetur ridiculus mus. Eget sit amet tellus cras adipiscing. Placerat duis ultricies lacus sed turpis tincidunt id aliquet. Vel pretium lectus quam id leo in. Tellus elementum sagittis vitae et leo duis ut diam. A diam maecenas sed enim ut sem viverra aliquet eget. Feugiat nisl pretium fusce id velit ut tortor. Et netus et malesuada fames. Tellus elementum sagittis vitae et leo duis ut diam. Sapien eget mi proin sed libero enim sed faucibus turpis.&nbsp;</span>-->
<!--          </p>-->
<!--        </div>-->
<!--      </div>-->
<!--    </section>-->
<!--    <section class="u-align-left u-clearfix u-palette-3-light-3 u-section-3" id="sec-8043">-->
<!--      <div class="u-clearfix u-sheet u-sheet-1"></div>-->
<!--    </section>-->
      <section class="u-clearfix u-palette-3-light-2 u-section-1" id="sec-0fbe">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <h3 class="u-text u-text-1">Strand bias analysis in NGS data</h3>
        <p class="u-text u-text-2">In recent years, next-generation sequencing (NGS) has brought significant progress to genomic research and diagnostics. However, NGS is
still prone to many errors, and one of them is strand bias. This page provides interactive results of strand bias analysis in datasets of three major sequencing technology companies: Illumina,
Pacific Biosciences and Oxford Nanopore Technologies.
        </p>
      </div>
    </section>
    <section class="u-align-left u-clearfix u-palette-3-light-3 u-section-2" id="sec-64ec">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="fr-view u-clearfix u-rich-text u-text u-text-1">
          <h2 style="text-align: left;">Strand Bias</h2>
          <p style="text-align: left;">
            <span style="line-height: 2.0;">
                Strand bias is a situation when one DNA strand is favoured over the other strand.
            Due to the varying amount of evidence favouring one gene variant compared to another, there is a possibility
            of an incorrect evaluation of the result. Knowing the correct frequencies of gene variants is crucial for
            population genotype analyses. It is also an important factor in medicine because frequencies of gene variants
            identify numerous genetic disorders. In theory, the incorrect evaluation of results could lead to inaccurate
                diagnosis and, therefore, wrong treatment.
         &nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">
                In bioinformatics, k-mers refer to all possible substrings of length k contained in the nucleotide sequence.
    Reverse complement of k-mer is created by constructing a complementary sequence to it and then reversing the
    sequence so that the last nucleotide becomes the first. Here we recognise strand bias as varying frequencies of
    the k-mers and their reverse complements. When both k-mer and its reverse complement have the same number of
    occurrences, it means this k-mer has no bias. However, when their frequencies vary, it means k-mer has strand bias.
    The higher the difference between k-mer and its reverse complement, the more concerning the strand bias is.

         &nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">
In the interactive plots, strand bias is represented by the ratio of less frequent (LF) of the k-mer and its complement
              towards the more frequent (MF) subtracted from 1 and converted into a percentage. The subtraction from 1
               is needed because a ratio represents how much the strings are ‘alike’, while the focus should be on how
              much they differ.
         &nbsp;</span>
          </p>
            <p style="text-align: center;">
            <span style="line-height: 2.0;">
                SB = [(1 − LF/MF) ∗ 100]&nbsp;</span>
          </p>
          <h2 style="text-align: left;">Results of analysis for different datasets</h2>
          <p style="text-align: left;">
            <span style="line-height: 2.0;">Morbi enim nunc faucibus a pellentesque sit amet porttitor. Mattis pellentesque id nibh tortor id aliquet lectus proin nibh. Molestie nunc non blandit massa enim nec dui. Felis imperdiet proin fermentum leo vel orci porta. Natoque penatibus et magnis dis parturient montes nascetur ridiculus mus. Eget sit amet tellus cras adipiscing. Placerat duis ultricies lacus sed turpis tincidunt id aliquet. Vel pretium lectus quam id leo in. Tellus elementum sagittis vitae et leo duis ut diam. A diam maecenas sed enim ut sem viverra aliquet eget. Feugiat nisl pretium fusce id velit ut tortor. Et netus et malesuada fames. Tellus elementum sagittis vitae et leo duis ut diam. Sapien eget mi proin sed libero enim sed faucibus turpis.&nbsp;</span>
          </p>
        </div>
      </div>
    </section>
    <section class="u-align-left u-clearfix u-palette-3-light-3 u-section-3" id="sec-8043">
      <div class="u-clearfix u-sheet u-sheet-1"></div>
    </section>
    
        <footer class="u-align-center u-clearfix u-footer u-palette-3-base u-footer" id="sec-274b"><div class="u-clearfix u-sheet u-sheet-1" align="center">
        <p class="u-small-text u-text u-text-variant u-text-1"><span>This webpage was created as a part of the FI MUNI project.<a class="u-link" href="https://www.muni.cz/en/research/projects/67479" >
      MUNI/33/0765/2022</a></span></p>
      </div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
      <p class="u-text">
        <span style="font-size:15px;">&copy; Alexandra Skyslakova  </span>&nbsp
      </p>
        <p>
          <span style="font-size:15px;"><i class="fa fa-envelope" style="font-size:15px;color:white"></i>  alexandra.skyslakova@gmail.com </span>&nbsp

        </p>
        <p>
          <a href="https://github.com/alex-skyslakova"><span style="font-size:15px;"><i class="fa fa-github" style="font-size:15px"></i>  alex-skyslakova</span>
</a>
        </p>
    </section>
  
</body></html>