<!DOCTYPE html>
<html style="font-size: 16px;" lang="en"><head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="keywords" content="Strand Bias, Results of analysis for different datasets, Datasets">
    <meta name="description" content="">
    <title>Home</title>
    <link rel="stylesheet" href="{{ url_for('static', filename='styles/nicepage.css')}}" media="screen">
<link rel="stylesheet" href="{{ url_for('static', filename='styles/Home.css')}}" media="screen">
    <script class="u-script" type="text/javascript" src="{{ url_for('static',filename='styles/jquery.js') }}" defer=""></script>
    <script class="u-script" type="text/javascript" src="{{ url_for('static',filename='styles/nicepage.js') }}" defer=""></script>
    <meta name="generator" content="Nicepage 4.19.3, nicepage.com">
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,500,500i,600,600i,700,700i,800,800i">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    
    

    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": ""
}</script>
    <meta name="theme-color" content="#478ac9">
    <meta property="og:title" content="Home">
    <meta property="og:type" content="website">
  </head>
  <body data-home-page="{{ url_for('home_page') }}" data-home-page-title="Home" class="u-body u-xl-mode" data-lang="en"><header class="u-clearfix u-header u-palette-3-light-3 u-header" id="sec-0286"><div class="u-clearfix u-sheet u-valign-middle u-sheet-1">

        <nav class="u-menu u-menu-one-level u-offcanvas u-menu-1" data-responsive-from="MD">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px; font-weight: 700; text-transform: uppercase;">
            <a class="u-button-style u-custom-active-border-color u-custom-active-color u-custom-border u-custom-border-color u-custom-borders u-custom-hover-border-color u-custom-hover-color u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-active-color u-custom-text-color u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg class="u-svg-link" viewBox="0 0 24 24"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg class="u-svg-content" version="1.1" id="menu-hamburger" viewBox="0 0 16 16" x="0px" y="0px" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg"><g><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</g></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-spacing-30 u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('home_page') }}" style="padding: 10px 0px;">Home</a>
</li><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('project_page') }}" style="padding: 10px 0px;">About Project</a>
</li><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('datasets_page') }}" style="padding: 10px 0px;">Datasets</a>
</li><li class="u-nav-item"><a class="u-border-2 u-border-active-palette-3-base u-border-grey-90 u-border-hover-palette-3-base u-border-no-left u-border-no-right u-border-no-top u-button-style u-nav-link u-text-active-palette-3-base u-text-grey-90 u-text-hover-palette-3-base" href="{{ url_for('analysis_page') }}" style="padding: 10px 0px;">Analysis</a>
            </li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-inner-container-layout u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('project_page') }}">About Project</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('project_page') }}">Datasets</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="{{ url_for('analysis_page') }}">Analysis</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header>
    <section class="u-clearfix u-palette-3-light-2 u-section-1" id="sec-0fbe">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <h2 class="u-text u-text-1">Strand bias analysis in NGS data</h2>
        <p class="u-text u-text-2">In recent years, next-generation sequencing (NGS) has brought significant progress to genomic research and diagnostics. However, NGS is
still prone to many errors, and one of them is strand bias. This page provides interactive results of strand bias analysis in datasets of three major sequencing technology companies: Illumina,
Pacific Biosciences and Oxford Nanopore Technologies.
        </p>
      </div>
    </section>
    <section class="u-align-left u-clearfix u-palette-3-light-3 u-section-2" id="sec-64ec">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="fr-view u-clearfix u-rich-text u-text u-text-1">
          <h3 style="text-align: left;">Strand Bias</h3>
          <p style="text-align: left;">
            <span style="line-height: 2.0;">
                Strand bias is a situation when one DNA strand is favoured over the other strand.
            Due to the varying amount of evidence favouring one gene variant compared to another, there is a possibility
            of an incorrect evaluation of the result. Knowing the correct frequencies of gene variants is crucial for
            population genotype analyses. It is also an important factor in medicine because frequencies of gene variants
            identify numerous genetic disorders. In theory, the incorrect evaluation of results could lead to inaccurate
                diagnosis and, therefore, wrong treatment.
         &nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">
                In bioinformatics, k-mers refer to all possible substrings of length k contained in the nucleotide sequence.
    Reverse complement of k-mer is created by constructing a complementary sequence to it and then reversing the
    sequence so that the last nucleotide becomes the first. Here we recognise strand bias as varying frequencies of
    the k-mers and their reverse complements. When both k-mer and its reverse complement have the same number of
    occurrences, it means this k-mer has no bias. However, when their frequencies vary, it means k-mer has strand bias.
    The higher the difference between k-mer and its reverse complement, the more concerning the strand bias is.

         &nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">
In the interactive plots, strand bias is represented by the ratio of less frequent (LF) of the k-mer and its complement
              towards the more frequent (MF) subtracted from 1 and converted into a percentage. The subtraction from 1
               is needed because a ratio represents how much the strings are ‘alike’, while the focus should be on how
              much they differ.
         &nbsp;</span>
          </p>
            <p style="text-align: center;">
            <span style="line-height: 2.0;">
                SB = [(1 − LF/MF) ∗ 100]&nbsp;</span>
          </p>
          <h3 style="text-align: left;">Results of analysis for different datasets</h3>
          <p style="text-align: left;">
            <span style="line-height: 2.0;">Analysis of strand bias in the datasets has shown that each technology
is affected by strand bias. Strand bias for k higher than seven starts to drastically increase, as
can be seen from the randomised dataset plot in Figure 4.1. Therefore,
in this section, I addressed to results gained for k in the range from 5
to 7, including. Table 4.1 shows strand bias results for this range for
one representative dataset of each technology.


&nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">Out of the three technologies I analysed, PacBio analysis resulted
in the lowest strand bias levels, with mean strand bias ranging from
1.86% to 2.14%. The second PacBio dataset had very similar results,
with a difference of at most 0.03% in both mean and median. The third
dataset presented slightly higher numbers. Mean strand biases for k
from 5 to 7 were 1.88%, 2.71% and 4.49%. &nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">Illumina presented slightly higher levels of strand bias, having
strand bias from 2.98% to 3.94% among a selected range of k. Two control datasets presented similar values with a difference of at most
0.3% from the first dataset in both mean and median.


&nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">Results of Nanopore analysis have shown significantly higher
strand bias than the previous technologies had. With strand bias levels
from 15.4 to 19.8 in the selected range of, it is more than three times
higher compared to Illumina. The second dataset containing three
runs presented slightly lower bias levels, with mean strand bias from
12.35% to 16.45%. However, third dataset resulted in strand bias in
range from 25.71% to 30.82%. Statistical significance and comparation
of results among representants of each technology using Statistica is
briefly commented in Appendix C.&nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">In conclusion, while results of PacBio and Illumina are relatively
small and homogenous among datasets, Nanopore presented strand
bias of much higher levels and different ranges among all three Nanopore
datasets.&nbsp;</span>
          </p>
            <p style="text-align: left;">
            <span style="line-height: 2.0;">&nbsp;</span>
          </p>
            <style type="text/css">
.tg  {border:none;border-collapse:collapse;border-color:#ccc;border-spacing:0;}
.tg td{background-color:#fff;border-color:#ccc;border-style:solid;border-width:0px;color:#333;
  font-family:Arial, sans-serif;font-size:14px;overflow:hidden;padding:5px 15px;word-break:normal;}
.tg th{background-color:#f0f0f0;border-color:#ccc;border-style:solid;border-width:0px;color:#333;
  font-family:Arial, sans-serif;font-size:14px;font-weight:normal;overflow:hidden;padding:5px 15px;word-break:normal;}
.tg .tg-3usy{background-color:##fef9e6;border-color:#f9e79b;text-align:center;vertical-align:middle}
.tg .tg-4ebj{background-color:#fef9e6;border-color:#f9e79b;text-align:center;vertical-align:middle}
.tg .tg-vaw9{background-color:#f9e79b;border-color:#f9e79b;text-align:left;vertical-align:middle}
.tg .tg-axn2{background-color:#f9e79b;border-color:#f9e79b;text-align:center;vertical-align:middle}
.tg .tg-bh5n{background-color:#f9e79b;border-color:#f9e79b;text-align:center;vertical-align:middle}
</style>
<table class="tg" align="center">
<thead>
  <tr>
    <th class="tg-vaw9">Dataset</th>
    <th class="tg-vaw9">\emph{k}</th>
    <th class="tg-vaw9">Mean</th>
    <th class="tg-vaw9">Median</th>
    <th class="tg-vaw9">Modus</th>
    <th class="tg-vaw9">5th pc.</th>
    <th class="tg-vaw9">95th pc.</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td class="tg-axn2">PacBio</td>
    <td class="tg-4ebj">5</td>
    <td class="tg-4ebj">1.86</td>
    <td class="tg-4ebj">1.22</td>
    <td class="tg-4ebj">0.07</td>
    <td class="tg-4ebj">0.14</td>
    <td class="tg-4ebj">4.33</td>
  </tr>
  <tr>
    <td class="tg-bh5n">PacBio</td>
    <td class="tg-3usy">6</td>
    <td class="tg-3usy">1.97</td>
    <td class="tg-3usy">1.36</td>
    <td class="tg-3usy">0.29</td>
    <td class="tg-3usy">0.14</td>
    <td class="tg-3usy">5.28</td>
  </tr>
  <tr>
    <td class="tg-axn2">PacBio</td>
    <td class="tg-4ebj">7</td>
    <td class="tg-4ebj">2.14</td>
    <td class="tg-4ebj">1.5</td>
    <td class="tg-4ebj">0</td>
    <td class="tg-4ebj">0.14</td>
    <td class="tg-4ebj">5.49</td>
  </tr>
  <tr>
    <td class="tg-bh5n">Illumina</td>
    <td class="tg-3usy">5</td>
    <td class="tg-3usy">2.98</td>
    <td class="tg-3usy">2.3</td>
    <td class="tg-3usy">0.014</td>
    <td class="tg-3usy">0.28</td>
    <td class="tg-3usy">7.47</td>
  </tr>
  <tr>
    <td class="tg-axn2">Illumina</td>
    <td class="tg-4ebj">6</td>
    <td class="tg-4ebj">3.46</td>
    <td class="tg-4ebj">2.63</td>
    <td class="tg-4ebj">2.29</td>
    <td class="tg-4ebj">0.26</td>
    <td class="tg-4ebj">9.2</td>
  </tr>
  <tr>
    <td class="tg-bh5n">Illumina</td>
    <td class="tg-3usy">7</td>
    <td class="tg-3usy">3.94</td>
    <td class="tg-3usy">2.94</td>
    <td class="tg-3usy">0</td>
    <td class="tg-3usy">0.27</td>
    <td class="tg-3usy">10.53</td>
  </tr>
  <tr>
    <td class="tg-axn2">Nanopore</td>
    <td class="tg-4ebj">5</td>
    <td class="tg-4ebj">15.4</td>
    <td class="tg-4ebj">12.35</td>
    <td class="tg-4ebj">22.8</td>
    <td class="tg-4ebj">1.08</td>
    <td class="tg-4ebj">42.77</td>
  </tr>
  <tr>
    <td class="tg-bh5n">Nanopore</td>
    <td class="tg-3usy">6</td>
    <td class="tg-3usy">17.85</td>
    <td class="tg-3usy">14.69</td>
    <td class="tg-3usy">4.1</td>
    <td class="tg-3usy">1.42</td>
    <td class="tg-3usy">47.37</td>
  </tr>
  <tr>
    <td class="tg-axn2">Nanopore</td>
    <td class="tg-4ebj">7</td>
    <td class="tg-4ebj">19.8</td>
    <td class="tg-4ebj">16.12</td>
    <td class="tg-4ebj">4.39</td>
    <td class="tg-4ebj">1.53</td>
    <td class="tg-4ebj">51.42</td>
  </tr>
</tbody>
</table>
        </div>
      </div>
    </section>
<!--    <section class="u-align-left u-clearfix u-palette-3-light-3 u-section-3" id="sec-8043">-->
<!--      <div class="u-clearfix u-sheet u-sheet-1"></div>-->
<!--    </section>-->
    
    
    <footer class="u-align-center u-clearfix u-footer u-palette-3-base u-footer" id="sec-274b"><div class="u-clearfix u-sheet u-sheet-1" align="center">
        <p class="u-small-text u-text u-text-variant u-text-1"><span>This webpage was created as a part of the FI MUNI project.<a class="u-link" href="https://www.muni.cz/en/research/projects/67479" >
      MUNI/33/0765/2022</a></span></p>
      </div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
      <p class="u-text">
        <span style="font-size:15px;">&copy; Alexandra Skyslakova  </span>&nbsp
      </p>
        <p>
          <span style="font-size:15px;"><i class="fa fa-envelope" style="font-size:15px;color:white"></i>  alexandra.skyslakova@gmail.com </span>&nbsp

        </p>
        <p>
          <a href="https://github.com/alex-skyslakova"><span style="font-size:15px;"><i class="fa fa-github" style="font-size:15px"></i>  alex-skyslakova</span>
</a>
        </p>
    </section>
  
</body></html>